lib/identity_key/Kconfig

```diff
@@ -11,6 +11,9 @@ menuconfig IDENTITY_KEY
 	depends on !TRUSTED_EXECUTION_NONSECURE
 	depends on MAIN_STACK_SIZE >= 2048
 	depends on ASSERT
+	select PSA_WANT_ALG_SHA_256
+	select PSA_WANT_ALG_ECDSA
+	select PSA_WANT_ECC_SECP_R1_192
```

Comment:
"We can create a read and write identity key config and then we can use that to only enable this when we write the key and not when we read it."

That sounds good. Or just select the readers dependencies and let the provisioning sample enable what it needs to write.

We definitely don't want to pull in features in production that are actually only needed in the provisioning image.

Reply:
Which feature is only needed when reading out the key?

Reply:
@SebastianBoe PR is updated, your comment made me check the initial attestation, and it was also failing.

I have also updated so that the identity key generation only requires the ECC key type to generate the key, and removed mention about ECDSA and SHA in there, because it is not really part of it.

Reply:
Great stuff!